luma/lcd/device.py

```diff
@@ -879,6 +879,131 @@ def command(self, cmd, *args):
             self._serial_interface.data(list(args))
 
 
+class ili9488(backlit_device, __framebuffer_mixin):
+    """
+    Serial interface to a 262k color (6-6-6 RGB) ILI9488 LCD display.
+
+    On creation, an initialization sequence is pumped to the display to properly
+    configure it. Further control commands can then be called to affect the
+    brightness and other settings.
+
+    :param serial_interface: the serial interface (usually a
+        :py:class:`luma.core.interface.serial.spi` instance) to delegate sending
+        data and commands through.
+    :param width: The number of pixels laid out horizontally.
+    :type width: int
+    :param height: The number of pixels laid out vertically.
+    :type height: int
+    :param rotate: An integer value of 0 (default), 1, 2 or 3 only, where 0 is
+        no rotation, 1 is rotate 90° clockwise, 2 is 180° rotation and 3
+        represents 270° rotation.
+    :type rotate: int
+    :param framebuffer: Framebuffering strategy, currently instances of
+        ``diff_to_previous()`` or ``full_frame()`` are only supported.
+    :type framebuffer: luma.core.framebuffer.framebuffer
+    :param bgr: Set to ``True`` if device pixels are BGR order (rather than RGB).
+    :type bgr: bool
+    :param h_offset: Horizontal offset (in pixels) of screen to device memory
+        (default: 0).
+    :type h_offset: int
+    :param v_offset: Vertical offset (in pixels) of screen to device memory
+        (default: 0).
+    :type v_offset: int
+
+    .. versionadded:: 2.11.0
+    """
+
+    def __init__(self, serial_interface=None, width=480, height=320, rotate=0,
+                 framebuffer=None, h_offset=0, v_offset=0, bgr=False, **kwargs):
+        super(ili9488, self).__init__(luma.lcd.const.ili9488, serial_interface, **kwargs)
+        self.capabilities(width, height, rotate, mode="RGB")
+        self.init_framebuffer(framebuffer, 25)
+
+        if h_offset != 0 or v_offset != 0:
+            def offset(bbox):
+                left, top, right, bottom = bbox
+                return (left + h_offset, top + v_offset, right + h_offset, bottom + v_offset)
+            self.apply_offsets = offset
+        else:
+            self.apply_offsets = lambda bbox: bbox
+
+        # Supported modes
+        supported = (width, height) in [(480, 320)]  # full
+        if not supported:
+            raise luma.core.error.DeviceDisplayModeError(
+                f"Unsupported display mode: {width} x {height}")
+
+        # RGB or BGR order
+        order = 0x00 if bgr else 0x08
+
+        # Initialization sequence, adapted from
+        #
+        # `https://github.com/birdtechstep/fbtft/blob/master/fb_ili9488.c`
```

Comment:
no need for backticks around the url here

Reply:
Removed backticks

Reply:
@wightmanjr the branch doesn't show any new commits though..